scripts/build/shims/at.js

```diff
@@ -10,9 +10,10 @@ const at = (isOptionalObject, object, index) => {
     return;
   }
 
-  return object.at
-    ? object.at(index)
-    : object[index < 0 ? object.length + index : index];
+  if (Array.isArray(object)) {
```

Comment:
I am wondering if we should include brand checking for safety...

Reply:
.at not only available on array, but we can check array and string only for now.